### ISSUE TITLE
Long input in ProjectsTable causes horizontal scrollbars to appear on /brigades

### ISSUE THREAD

**Issue by rkaufman13** [codeforamerica/brigade-project-index-statusboard#128]
**Describe the bug**
If the project description contains especially long words (mostly URLs), the table shown on /brigades appears with a horizontal scrollbar.

**To Reproduce**
1. Go to https://projects.brigade.network/brigades
2. Select Milwaukee Data Initiative
3. View scrollbar appear, caused by the mpd-call-log-gscript project containing a 76-character URL

**Expected behavior**
Expected that text would fit within table

**Desktop (please complete the following information):**
 - OS: MacOS 10.13 (11" macbook air, the small screen size of which may explain why the table does not have a lot of wiggle room)
 - Browser: Chrome

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Nice spotted, thank you for the report!

It is not clear to me what is "correct" behavior, but it definitively shouldn't behave like that. Additionally, comparing the table width between Milwaukee Data Initiative and BetaNYC shows very different line breaking. I can't guess why this is happening, but if it happens on `/brigades` it very likely happens on `/projects` as well.

Probably references #80, #42, #51, and #44.

**Comment by rkaufman13** [comment]
You're correct, it does cause horizontal scrolling on both pages. But now I'm also less sure of what is causing this bug than I was over the weekend. I thought it was simply caused by there being a long unbroken string of characters in the table, but there is not a similar issue in the Projects column despite them having as far as I can tell, the same styling. See attached screenshot of the Birmingham projects list, where the project name wraps correctly.
![Screen Shot 2021-08-30 at 5 37 45 PM](https://user-images.githubusercontent.com/16939591/131409436-a0f98abb-3fdb-43c2-be14-94fdcac617b3.png)

I'm rather mystified!

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
I am not a webdeveloper, but it could relate to the longest available string that cannot be line-broken? E.g., `http://itmdapps.milwaukee.gov/MPDCallData/currentCADCalls/callsService.faces` in the case of Milwaukee Data Initiative.

In my subjective opinion, it makes sense that `/projects` is allowed to have scrollbars for the x-axis since it has to display a lot of columns, but perhaps there is a better option for `/brigades`.

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
> See attached screenshot of the Birmingham projects list, where the project name wraps "correctly." (It looks...awful, but at least the table isn't stretched out!)

Wow, I got too caught up in manually adding [word-wrap](https://www.w3schools.com/cssref/css3_pr_word-wrap.asp) and forgot that part, sorry about that! In the end, I believe this is the "culprit" for the different behavior in the Projects column

https://github.com/codeforamerica/brigade-project-index-statusboard/blob/8feacf5a55d08bfaad4334c38004a1f70d5cf467/statusboard/src/components/ProjectsTable/ProjectsTable.scss#L21-L23

**Comment by rkaufman13** [comment]
Proposal: fix CSS by changing word-break (which only works, apparently, with words that have spaces between them or Chinese/Japanese/Korean text...who knew) to word-wrap: break-word (or even better, overflow-wrap, [which is the new name for word-wrap](https://www.w3.org/TR/css-text-3/#overflow-wrap))....and then I can create a new issue for how the table looks on mobile :)

<img src="https://user-images.githubusercontent.com/16939591/132004855-0f3cf581-067a-4167-80fb-7387cb0a348d.png" width="300">
<img src="https://user-images.githubusercontent.com/16939591/132004858-d5e2fdaa-4aca-4c81-b0c0-cc293098741c.png" width="300">

I haven't worked with it at all but there seems to be a plugin called [React-super-responsive-table](https://www.npmjs.com/package/react-super-responsive-table) that looks great, at least in theory, for uses like this, if you agree. The drawback would be that on small screens the data is no longer technically presented as tabular, but my semi-strongly-held opinion is that data should be tabular if users need to compare between rows (think census data over time for example) and otherwise the need for it to be in a proper table is less important. As I'm thinking out loud maybe my mental model for tabular data is "could it be graphed?"

**So in summary,** if it's OK with you, I'll move forward by tweaking the CSS as a stopgap fix, and create a new issue or find an existing issue about the mobile table display, where we can take the discussion about the best way to move forward to make the table mobile friendly, if at all.

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
> I haven't worked with it at all but there seems to be a plugin called React-super-responsive-table that looks great,

I actually don't know. I have mostly kept to `react-table` because it was already there and it uses native HTML components (as suggested in e.g. #43). I've also considered digging into Material-UI both for their table support and that #74/#81 are likely going to benefit from having an easy-to-use tree. I am sorry if I made things more confusing not less :/

> if it's OK with you, I'll move forward by tweaking the CSS as a stopgap fix

Sounds good!

> and create a new issue or find an existing issue about the mobile table display, where we can take the discussion about the best way to move forward to make the table mobile friendly,

I think the best matching existing issue is #51, which is a bit outdated. Opening a new issue regarding the table, in particular, sounds better to me.

Thank you!

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Regarding the `/projects` page, I'd argue there is a purpose to keeping all the columns even on small devices. But at the very least it would benefit from having a line-wrapping and/or narrower "Project" column.

In my opinion, the purpose of `https://projects.brigade.network/` and `https://projects.brigade.network/projects` should be to present quite a bit of information about Code for America project and the brigades and a lot of flexibility in filtering/sorting (at least in the future 😛 ). There are different opinions about what should go where (and how many more pages there should be (I want a `/all_projects` page ^^) but I am definitively open to suggestion and I expect it to be a little bit of discussion on the Zoom meeting on Wednesday

**Comment by rkaufman13** [comment]
Closed by #131
